modules/serverless-creating-broker.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * serverless-using-brokers.adoc
+
+[id="serverless-creating-broker_{context}"]
+= Creating a broker
+
+To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+
+.Prerequisites
+* Knative Eventing installed, which includes the `ClusterRole`.
+
+.Procedure
+. Create the `ServiceAccount` objects:
++
+----
+$ oc -n default create serviceaccount eventing-broker-ingress
+$ oc -n default create serviceaccount eventing-broker-filter
+----
+
+. Give those objects RBAC permissions:
++
+----
+$ oc -n default create rolebinding eventing-broker-ingress \
+  --clusterrole=eventing-broker-ingress \
+  --serviceaccount=default:eventing-broker-ingress
+$ oc -n default create rolebinding eventing-broker-filter \
+  --clusterrole=eventing-broker-filter \
+  --serviceaccount=default:eventing-broker-filter
+----
+
+. Create RBAC permissions granting access to shared configmaps for logging, tracing, and metrics configuration (optional):
++
+----
+$ oc -n knative-eventing create rolebinding eventing-config-reader-default-eventing-broker-ingress \
+  --clusterrole=eventing-config-reader \
+  --serviceaccount=default:eventing-broker-ingress
+$ oc -n knative-eventing create rolebinding eventing-config-reader-default-eventing-broker-filter \
+  --clusterrole=eventing-config-reader \
+  --serviceaccount=default:eventing-broker-filter
```

Comment:
@abrennan89  @matzew  So, should we have these at all?  IIUC, the "configmappropagations" mechanism should replicate the configmaps into the users' namespace, so we don't need these at all, even optionally, right?

Reply:
ok, that is ture and we now ensured that the `configmappropagations` works on OCP

@abrennan89  so, I agree here - let's get rid of them